### ISSUE TITLE
[Bug] Alt + F Freeze

### ISSUE THREAD

**Issue by starwarsfate** [LiveEnhancementSuite/LESforWindows#12]
The Alt + F and Shift + Alt + F new shortcuts are not working.

**Comment by Inversil** [comment]
yeah, I realized I made the same mistake as with https://github.com/LiveEnhancementSuite/LESforWindows/issues/8
I assumed everyone was running their live in English, so currently if you use a different language these shortcuts don't work...

**Comment by Inversil** [comment]
fixed here!
feel free to clone the git and run the .ahk to try see if it works!
https://github.com/LiveEnhancementSuite/LESforWindows/commit/7f1819e80a88051928a1e2b55395cb6557880818

**Comment by krpna** [comment]
Hey brother, its me here commenting with another account. Thanks for your nice work
Its now working if I use ableton without fullscreen, but if I use fullscreen with the F11 key this shortcut does not work.
